examples/zephyr/README.md

```diff
@@ -1,34 +1,118 @@
+currently designated as "beta". We will update this page when this port
+is ready for general availability. For production devices, please use
+the [Golioth Zephyr SDK](https://github.com/golioth/golioth-zephyr-sdk).
 
-### Install Zephyr SDK
+The SDK contains a set of examples designed to run using Zephyr
+RTOS.
 
-Follow [Zephyr Getting Started Guide](https://docs.zephyrproject.org/latest/develop/getting_started/).
+API documentation: <https://firmware-sdk-docs.golioth.io/>
 
-### Setup west project
+## Install Golioth Firmware SDK with Zephyr RTOS Support
 
-``` sh
-cd ~/golioth-workspace
+### Requirements:
+
+* Install the latest version of the [Zephyr
+  SDK](https://github.com/zephyrproject-rtos/sdk-ng/releases/tag/v0.16.1).
+* Install [Python](https://www.python.org/downloads/)
+* Install [the West
+  tool](https://docs.zephyrproject.org/latest/develop/west/install.html)
+
+
+Most platforms are already supported with mainline [Zephyr
+RTOS](https://www.zephyrproject.org/). This repository can be added to
+any Zephyr based project as new
+[west](https://docs.zephyrproject.org/3.4.0/develop/west/index.html)
+module. However, for making things simple, this repository can also
+serve as
+[west](https://docs.zephyrproject.org/3.4.0/develop/west/index.html)
+manifest repo.
+
+### Using the Golioth Firmware SDK as a Zephyr manifest repository
+
+Execute this command to download this repository together with all
+dependencies:
+
+``` console
 west init -m https://github.com/golioth/golioth-firmware-sdk.git --mf west-zephyr.yml
 west update
 cd modules/lib/golioth-firmware-sdk && git submodule update --init --recursive
 ```
 
-### Configure PSK and PSK-ID credentials
+### Adding the Golioth Firmware SDK to an existing Zephyr west project
 
-Add credentials to ``~/golioth-workspace/modules/lib/golioth-firmware-sdk/examples/zephyr/golioth_basics/prj.conf``
-file:
+Alternatively, add the following entry to the `west.yml` file of an
+existing
+[west](https://docs.zephyrproject.org/3.4.0/develop/west/index.html)
+based project (e.g. Zephyr RTOS):
 
-``` cfg
-CONFIG_GOLIOTH_SAMPLE_PSK_ID="my-psk-id@my-project"
-CONFIG_GOLIOTH_SAMPLE_PSK="my-psk"
+``` yaml
+# Golioth repository.
+- name: golioth
+  path: modules/lib/golioth-firmware-sdk
+  revision: main
+  url: https://github.com/golioth/golioth-firmware-sdk.git
+  import: west-external.yml
+```
+
+and clone all repositories including that one by running:
+
+``` console
+west update
+cd modules/lib/golioth-firmware-sdk && git submodule update --init --recursive
 ```
 
-### Build and run sample project
+Follow [Zephyr Getting
+Started](https://docs.zephyrproject.org/3.4.0/develop/getting_started/index.html)
+for details on how to setup Zephyr based projects.
 
-``` sh
-west build -b qemu_x86 modules/lib/golioth-firmware-sdk/examples/zephyr/golioth_basics
-west build -t run
+## Using Golioth with Nordic's nRF Connect SDK
+
+Platforms like [nRF9160
```

Comment:
I don't know that we need this sentence, but if we want an introduction here I think making it part agnostic makes sense (or at least refer to the part and not the board). Maybe something like:

Nordic maintains a fork of the Zephyr project with additional proprietary functionality for their silicon, such as cellular connectivity.

Reply:
I think think you're right that we don't need this and have removed it in the latest push.